### ISSUE TITLE
Windows 10 x32 not compatible 

### ISSUE THREAD

**Issue by code9ster** [balena-io/balena-cli#1280]
- **10.17.4**
- **Standalone install:** yes
- **Node version:** no idea what this is
- **Npm version:** no idea what this is
- **windows 10 32 bit**
    I setup system env, path to c:\balena-cli.  balena ci folder is in root of C:.  when I type balena in cmd.exe,  I get this version of C:\balena-cli\balena.exe is not compatible with the version of windows you're running.  The balena cli I downloaded said it is compatible with x32 and x64 am I missing something?

---

*Please keep in mind that we try to use the issue tracker of this repository for specific bug reports & CLI feature requests. General & troubleshooting questions are encouraged to be posted to the [balena forums](https://forums.balena.io) where the community can both contribute and benefit from the answers.*

*Before submitting this issue please check that this issue is not a duplicate. If there is another issue describing the same problem or feature please add your information to the existing issue's comments.*

**Comment by pdcastro** [comment]
I can confirm that `"balena-cli-v10.17.4-win32-x64.zip"` is only expected to work on 64-bit OS / processors. I agree that, unfortunately, the "win32" term makes it sound like the file is compatible with 32-bit architectures; the explanation is that "win32" is the value returned by `process.arch` (Node.js) on Windows, whether 32 or 64 bits. We are renaming it to "windows" in the upcoming CLI v11, to avoid this confusion.

A workaround that is expected to work on 32-bit computers is to install the CLI via the [NPM installation method](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md#npm-installation). It is more complex and normally only recommended for Node.js developers who are familiar with 'npm', but it is the only 32-bit option at the moment.

As far as I am aware, we were not producing 32-bit executables because we were not aware of "anyone who might need them", but this issue may serve as a counter example -- so thank you for raising it. We will consider the required changes / implications to produce 32-bit executables.

**Comment by pdcastro** [comment]
I am closing this issue in favour of #1489.  An improvement that has been made in v11.x.x is to rename the downloadable installers to at least make it clearer that they are 64-bit installers, removing "win32" from the name:

&nbsp; | File name
-- | --
Before: | `balena-cli-v10.17.4-win32-x64.zip`
After:   | `balena-cli-v11.17.1-windows-x64-standalone.zip`
&nbsp; | `balena-cli-v11.17.1-windows-x64-installer.exe`
